go/types.go

```diff
@@ -73,13 +74,12 @@ func (v Version) String() string {
 	}
 }
 
-// CT Version constants, see section 3.2 of the RFC.
+// CT Version constants from section 3.2.
 const (
 	V1 Version = 0
 )
 
-// SignatureType differentiates STH signatures from SCT signatures, see RFC
-// section 3.2
+// SignatureType differentiates STH signatures from SCT signatures, see section 3.2
```

Comment:
full stop at end?  I only mention it because you've been so meticulous about adding them in other places :)